src/main/java/org/fogbowcloud/saps/engine/core/dispatcher/restlet/DatabaseApplication.java

```diff
@@ -31,283 +32,179 @@
-	 * @param priority                 priority of new tasks
-	 * @param email                    user email
-	 */
-	public void addNewTasks(String lowerLeftLatitude, String lowerLeftLongitude, String upperRightLatitude,
-			String upperRightLongitude, Date initDate, Date endDate, String inputdownloadingPhaseTag,
-			String preprocessingPhaseTag, String processingPhaseTag, String priority, String email) {
-		submissionDispatcher.addNewTasks(lowerLeftLatitude, lowerLeftLongitude, upperRightLatitude, upperRightLongitude,
-				initDate, endDate, inputdownloadingPhaseTag, preprocessingPhaseTag, processingPhaseTag,
-				Integer.parseInt(priority), email);
-	}
-
-	/**
-	 * This function add new user in Catalog.
-	 * 
-	 * @param userEmail  user email
-	 * @param userName   user name
-	 * @param userPass   user password
-	 * @param userState  user state
-	 * @param userNotify user notify
-	 * @param adminRole  administrator role
-	 * @throws SQLException
-	 */
-	public void createUser(String userEmail, String userName, String userPass, boolean userState, boolean userNotify,
-			boolean adminRole) throws SQLException {
-		submissionDispatcher.addUser(userEmail, userName, userPass, userState, userNotify, adminRole);
-	}
-
-	/**
-	 * This function gets user information in Catalog.
-	 * 
-	 * @param userEmail user email
-	 * @return saps user
-	 */
-	public SapsUser getUser(String userEmail) {
-		return submissionDispatcher.getUser(userEmail);
-	}
-
-	/**
-	 * This function search all processed tasks from area (between latitude and
-	 * longitude coordinates) between initial date and end date with
-	 * inputdownloading, preprocessing and processing tags.
-	 * 
-	 * @param lowerLeftLatitude        lower left latitude (coordinate)
-	 * @param lowerLeftLongitude       lower left longitude (coordinate)
-	 * @param upperRightLatitude       upper right latitude (coordinate)
-	 * @param upperRightLongitude      upper right longitude (coordinate)
-	 * @param initDate                 initial date
-	 * @param endDate                  end date
-	 * @param inputdownloadingPhaseTag inputdownloading phase tag
-	 * @param preprocessingPhaseTag    preprocessing phase tag
-	 * @param processingPhaseTag       processing phase tag
-	 * @return processed tasks list following description
-	 */
-	public List<SapsImage> searchProcessedTasks(String lowerLeftLatitude, String lowerLeftLongitude,
-			String upperRightLatitude, String upperRightLongitude, Date initDate, Date endDate,
-			String inputdownloadingPhaseTag, String preprocessingPhaseTag, String processingPhaseTag) {
-		return submissionDispatcher.searchProcessedTasks(lowerLeftLatitude, lowerLeftLongitude, upperRightLatitude,
-				upperRightLongitude, initDate, endDate, inputdownloadingPhaseTag, preprocessingPhaseTag,
-				processingPhaseTag);
-	}
+    private static final String DB_WEB_STATIC_ROOT = "./dbWebHtml/static";
+
+    public static final Logger LOGGER = Logger.getLogger(DatabaseApplication.class);
```

Comment:
general comment: doc-it obvious stuff is a waste of time. I'd delete any obvious java-doc

Reply:
Added fixme.